src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Configuration/AppSettings.cs

```diff
@@ -1,7 +1,6 @@
-using System;
 using System.Collections.Generic;
-using Altinn.App.Services.Helpers.Extensions;
 using Altinn.App.Services.Models;
```

Comment:
We have quite many classes still using the App.Services namespace. Should we create an issue to move all of these to the platformServices namespace?

Reply:
Created new issue.